frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/utils.ts

```diff
@@ -5,9 +5,11 @@ import { PrometheusResponse, DataPoint } from '@console/internal/components/grap
 export const getMetricType = (resource, metricType) => _.get(resource, ['metric', metricType], '');
 
 export const getGraphVectorStats: GetStats = (response, metricType, unit) => {
-  return response.map((r) => {
+  return response.map((r, i) => {
+    const name = getMetricType(r, metricType);
```

Comment:
Please add the type for the passed parameters inside `getMetricType` function.

Reply:
ack